server/msteams/client.go

```diff
@@ -472,12 +493,29 @@ func (tc *ClientImpl) SubscribeToChats(baseURL string, webhookSecret string, pay
 	return tc.subscribe(baseURL, webhookSecret, resource, changeType)
 }
 
-func (tc *ClientImpl) RefreshSubscription(subscriptionID string) error {
+func (tc *ClientImpl) SubscribeToUserChats(userID, baseURL string, webhookSecret string, pay bool) (*Subscription, error) {
+	resource := fmt.Sprintf("/users/%s/chats/getAllMessages", userID)
+	if pay {
+		resource = "chats/getAllMessages?model=B"
+	}
```

Comment:
This doesn't seem right. If we're using the metered API and the resource "user chat change notifications" doesn't support license modeling, then we shouldn't change the resource to "chats/getAllMessages" I think.

Reply:
good catch. Thanks